src/content/getusermedia/getdisplaymedia/index.html

```diff
@@ -25,25 +25,26 @@
   <link rel="icon" sizes="192x192" href="../../../images/webrtc-icon-192x192.png">
   <link href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet" type="text/css">
   <link rel="stylesheet" href="../../../css/main.css">
+  <link rel="stylesheet" href="css/styles.css">
+  <script src="https://unpkg.com/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
```

Comment:
This seems like an unnecessary dependency that shouldn't have been merged IMHO. Doesn't even load on Edge anymore.

I'll file an issue to revert it.

Reply:
We've had several disucssions around barebone vs contexualised samples before. The feedback I got from other web develoeprs were that the contexualised samples are more appreciated, as it shows how to use it in a real application. 

If it doesn't work in Edge we'll fix that. It's the only browser I haven't been able to test in.